lib/core/network/api_service.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: equal_keys_in_map
```

Comment:
Why?

Reply:
This moved to givt_repository.dart. Its because of the way the backend expects the params.
```
final params = {
      'OrderType': '3',
      'GroupType': '2',
      'FromDate': fromDate,
      'TillDate': tillDate,
      'TransactionStatusses': '1',
      'TransactionStatusses': '2',
      'TransactionStatusses': '3',
    };
```
It does not take a list on TransactionStatusses, so there have to be equal keys in the map to pass multiple statusses (Entered, to be processed, processed). If i don't add the linter marks the file with a warning and its very distracting & annoying. Its not a hard need to have that, but a strong want 🙏